dbt_project.yml

```diff
@@ -26,7 +26,10 @@ vars:
   snowplow_unified:
     # See https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/unified/ for more information and an interactive tool to help you with the variable setup
     # Please only add those that you change the values of to your root dbt_project.yml file, do not copy all values as this can lead to unexpected issues
-
+    
```

Comment:
I would default this to false and explicitly enable it within the integration_tests's project.yml so that the automated tests would run and then no need to revert anything later.